waterlog/src/Components/TankGraph.js

```diff
@@ -14,12 +14,12 @@ const TankGraph = (props) => {
   let forecast = [];
   for(let i = startTime; i <= endTime; i = i + 3600){
     forecast.push(1*i + 80);
-  }*/
+  }
   
   let data = {
-    labels: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'],
+    labels: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12','13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23', '24'],
```

Comment:
What is this for?

Reply:
it was for putting labels in the graph (x axis )   which must end up to 24 not 12

Reply:
I have remove  it was for testing purposes now its get data in the database